webpack.common.js

```diff
@@ -9,6 +9,15 @@ module.exports = {
         test: /\.tsx?$/,
         use: 'ts-loader',
         exclude: /node_modules/
+      },
+      {
+        test: /\.tsx?$/,
+        loader: 'string-replace-loader',
+        options: {
+          search: '{{VERSION}}',
+          replace: npmPackage.version,
```

Comment:
we could probably also inline this: `require("./package.json").version`

Reply:
We could but I prefer as it is. (Have imports / requires on top of files)